### PR TITLE
Fixed 'Bingtranslate' Typo #151

### PR DESCRIPTION
**Pull request by sarcastic22101**
Fixed 'bingtranslate' to 'bingtranslator' and added the new icon on the modal.html file #151 

#hacktoberfest

### CODE REVIEW

**Comment by sarcastic22101** [comment]
Also updated modal.html to have title attributes for each button. #144 

#hacktoberfest

**Comment by Ryuno-Ki** [comment]
Nice!
The icing on the cake would be a removal of the old Bing Translator icon :-)

File: modal/modal.html [hunk 1]
```diff
@@ -9,142 +9,142 @@
...
         </button>
 
-        <button class="col share clearfix" id="scoopit"
+        <button class="col share clearfix" id="scoopit" title=“Scoop.it”
                 data-share="https://www.scoop.it/bookmarklet?url">
             <img src="icons/scoopit.png" />
         </button>
 
-        <button class="col share clearfix" id="bingtranslate"
+        <button class="col share clearfix" id="bingtranslator” title=“Bing Translator”
```
Comment (Mte90) [inline modal/modal.html:105]: If you change the id, you need to to the same in the options.html or the settings won't match.

File: modal/modal.html [hunk 2]
```diff
@@ -9,142 +9,142 @@
 <body>
 <div id="share-directory">
     <div id="share-directory-grid" class="clearfix">
-        <button class="col share clearfix" id="facebook" data-share="https://www.facebook.com/sharer.php?u">
+        <button class="col share clearfix" id="facebook" title=“Facebook” data-share="https://www.facebook.com/sharer.php?u">
```
Comment (Mte90) [inline modal/modal.html:12]: Button tag doesn't have any `title` parameter, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button doesn't need that.

**Review by Mte90 — CHANGES_REQUESTED** [review]
I have to reject this pr as includes a lot of errors (title attributes doesn't exists in button tag) and the id changed is missing also in the other file.
Doesn't fit at all in the hacktoberfest for me.

File: modal/modal.html (hunk 2 above)
Comment (Ryuno-Ki) [reply modal/modal.html:12]: I'd add it as `alt` attribute to the `` within?

Comment (Mte90) [reply modal/modal.html:12]: Can get more sense to img tag instead of button, but is something that can be useful?

Comment (Ryuno-Ki) [reply modal/modal.html:12]: Of course!
Right now, the `<button>`s wouldn't be announced in a meaningful way to screenreaders.

**Pull request closed without merging**